packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/rules/edit/routing.js

```diff
@@ -0,0 +1,53 @@
+import { TRACKING_NAME } from '../../../../../constants';
+import { TRACKING_SUFFIX as LISTENERS_TRACKING_SUFFIX } from '../../../../constants';
+import { TRACKING_SUFFIX as POLICIES_TRACKING_SUFFIX } from '../../constants';
+import { TRACKING_SUFFIX as L7_RULES_TRACKING_SUFFIX } from '../constants';
+import { TRACKING_SUFFIX } from './constants';
```

Comment:
Didn't you move all tracking data in one global file ?

Reply:
I will do it on the other PR, but I'm still not able to make the tracking inside the banner working so I wen back to this task